backends/rapidpro/backend.go

```diff
@@ -379,12 +379,12 @@ func (b *backend) WriteChannelEvent(ctx context.Context, event courier.ChannelEv
 	return writeChannelEvent(timeout, b, event)
 }
 
-// WriteChannelLogs persists the passed in logs to our database, for rapidpro we swallow all errors, logging isn't critical
-func (b *backend) WriteChannelLogs(ctx context.Context, logs []*courier.ChannelLog) error {
+// WriteChannelLog persists the passed in log to our database, for rapidpro we swallow all errors, logging isn't critical
```

Comment:
naming here is intentional because `courier.ChannelLogger` will replace `courier.ChannelLog` and be called that